### DISCUSSION TITLE
Aggregated feed not working

### DISCUSSION THREAD

**Discussion by thoresson** [FreshRSS/FreshRSS#6035] [Q&A] [answered]
Hi!

I'm just getting started with a self-hosted installation of FreshRSS. I would like to use the aggregated RSS feed for all starred posts, but I get an error that the feed isn't valid. To my eye, it looks like there is an extra line break on the first line of the XML file. 

<img width="1475" alt="CleanShot 2024-01-11 at 20 14 05@2x" src="https://github.com/FreshRSS/FreshRSS/assets/6024521/b6fc46fe-c3cb-41a7-9f1f-2be51dd467cf">

Is this why the link is not recognized as a RSS feed?

Thanks,
Anders

**Comment by Alkarex** [comment]
No, that looks fine. You can try to paste your output into an XML validator. For instance online https://www.xmlvalidation.com

**Reply by thoresson** [reply]
It validated without any issues. 

I tried to add the feed to FreshRSS, just to check what the log would say: "cURL error 22: The requested URL returned error: 404". Can't understand why it gets a 404 when I can load the XML content by pasting the URL into the address field of my browser?

**Reply by thoresson** [reply]
But an 403 when using a browser where I'm not logged into FreshRSS. 

<img width="1383" alt="CleanShot 2024-01-11 at 22 12 17@2x" src="https://github.com/FreshRSS/FreshRSS/assets/6024521/6669d088-5d94-4ee3-b2b8-a5ae5021ae04">

**Reply by Frenzie** [reply]
Something I occasionally mess up is that I forget to quote `&` when pasting URLs to curl. Which is to say, are you use the `&user` and `&token` are in the URL?

**Reply by thoresson** [reply]
This is what the generated url looks like: https://<mydomain>/i/?a=rss&rid=65a05c21e87ee&hours=168

And just to make sure, this is where I get the link:

![CleanShot 2024-01-11 at 22 23 43@2x](https://github.com/FreshRSS/FreshRSS/assets/6024521/00b0bf18-6bf3-41db-b027-1b4abd283a64)

**Reply by Alkarex** [reply, marked as answer]
To be able to access that from another location, you need to setup a token in your user settings.

**Reply by thoresson** [reply]
Ah, thanks!
